common/src/WooCommerce.php

```diff
@@ -312,10 +312,15 @@ private function processReviews(\SimpleXMLElement $reviews): void {
                 $comment_data['comment_author_email'],
                 (int) $review->review_id
             );
+            $deleted = (int) $review->deleted;
```

Comment:
This should be `(bool)`, I believe.

Reply:
when it's bool it always returns `true` 
```$feed->addChild('deleted', false);``` results in `<deleted/>`

Reply:
Ah, I see.  Yeah, because it's not a string.  Fuck SimpleXML.